src/applications/vaos/components/ConfirmationDirectScheduleInfo.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import moment from 'moment';
+import moment from '../utils/moment-tz.js';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import FacilityAddress from './FacilityAddress';
+import AddToCalendar from './AddToCalendar';
+import { getFacilityAddress } from '../utils/appointment';
+import {
+  getTimezoneAbbrBySystemId,
+  getTimezoneBySystemId,
+} from './../utils/timezone';
 import { PURPOSE_TEXT } from '../utils/constants';
 
 export default function ConfirmationDirectScheduleInfo({
   data,
   facilityDetails,
   clinic,
   pageTitle,
+  appointmentLength,
 }) {
+  const dateTime = data.calendarData.selectedDates[0].datetime;
+  const timezone = getTimezoneBySystemId(data.vaSystem);
+  const momentDate = timezone
```

Comment:
Was this date wrong before? It seems like this is going to change the date shown in the header.

Reply:
This date was not wrong per se, but it did not have any timezone/offset attached to it, so when I generated an ICS, it would add the event in my local timezone, instead of the system timezone (Mountain Time).  Maybe instead of adding the timezone here, I should do it on the calendar page itself.

Reply:
I decided against adding the timezone to the slots themselves on the Calendar page because the API is expecting the offset to be +00:00 and I'd prefer not to mess with that.  Leaving it as is.